non-standard.json

```diff
@@ -14,6 +14,7 @@
   "org.scala-lang scalap":                           "version",
   "amplab spark-indexedrdd":                         "pom",
   "me.leaf akka-persistence-android":                "pom",
+  "com.typesafe config":                             "pom",
```

Comment:
I'm on it :-) https://github.com/scalacenter/scaladex/pull/305 Let's discuss the rest there